docs/configure_https.md

```diff
@@ -26,85 +28,61 @@ Otherwise, if you use IP address to connect your registry host, CN can be anythi
   cd ..
- ```
+```
 If you're using FQDN like **reg.yourdomain.com** to connect your registry host, then run this command to generate the certificate of your registry host:
+
 ```
   openssl ca -in yourdomain.com.csr -out yourdomain.com.crt -cert ca.crt -keyfile ca.key -outdir .
 ```
-If you're using **IP** to connect your registry host, you may instead run the command below:
+If you're using **IP**, say **192.168.1.101** to connect your registry host, you may instead run the command below:
+
 ```
-  
-  echo subjectAltName = IP:your registry host IP > extfile.cnf
+  echo subjectAltName = IP:192.168.1.101 > extfile.cnf
 
   openssl ca -in yourdomain.com.csr -out yourdomain.com.crt -cert ca.crt -keyfile ca.key -extfile extfile.cnf -outdir .
 ```
-##Configuration of Nginx
-After obtaining the **yourdomain.com.crt** and **yourdomain.com.key** files, change the directory to make/config/nginx in Harbor project.
-```
-  cd make/config/nginx
-```
-Create a new directory cert/, if it does not exist. Then copy **yourdomain.com.crt** and **yourdomain.com.key** to cert/, e.g. :
-```
-  cp yourdomain.com.crt cert/
-  cp yourdomain.com.key cert/ 
-```
+##Configuration and Installation
+After obtaining the **yourdomain.com.crt** and **yourdomain.com.key** files, 
+you can put them into directory such as ```/root/cert/```:
 
-Rename the existing configuration file of Nginx:
-```
-  mv nginx.conf nginx.conf.bak
-```
-Copy the template **nginx.https.conf** as the new configuration file:
 ```
-  cp nginx.https.conf nginx.conf
+  cp yourdomain.com.crt /root/cert/
+  cp yourdomain.com.key /root/cert/ 
 ```
-Edit the file nginx.conf and replace two occurrences of **harbordomain.com** to your own host name, such as reg.yourdomain.com . If you use a customized port rather than the default port 443, replace the port "443" in the line "rewrite ^/(.*) https://$server_name:443/$1 permanent;" as well. Please refer to the [installation guide](https://github.com/vmware/harbor/blob/master/docs/installation_guide.md) for other required steps of port customization.  
-```
-  server {
-    listen 443 ssl;
-    server_name harbordomain.com;
 
-    ...
-    
-  server {
-    listen 80;
-    server_name harbordomain.com;
-    rewrite ^/(.*) https://$server_name:443/$1 permanent;
-```
-Then look for the SSL section to make sure the files of your certificates match the names in the config file. Do not change the path of the files.
-```
-    ...
-    
-    # SSL
-    ssl_certificate /etc/nginx/cert/yourdomain.com.crt;
-    ssl_certificate_key /etc/nginx/cert/yourdomain.com.key;
-```
-Save your changes in nginx.conf.
 
-##Installation of Harbor
-Next, edit the file make/harbor.cfg , update the hostname and the protocol:
+Next, edit the file make/harbor.cfg , update the hostname and the protocol, and update the attributes ```ssl_cert``` and ```ssl_cert_key```:
+
 ```
   #set hostname
   hostname = reg.yourdomain.com
   #set ui_url_protocol
   ui_url_protocol = https
+  ......
```

Comment:
better add the line:
#The path of cert and key files for nginx, they are applied only the protocol is set to https